spec/policies/user_policy_spec.rb

```diff
@@ -33,6 +33,9 @@
     it { is_expected.not_to permit_action(:update) }
     it { is_expected.not_to permit_action(:edit) }
     it { is_expected.not_to permit_action(:destroy) }
+    it { expect(subject.send(:admin?)).to be false }
```

Comment:
@LucasAnderson1984 so it would seem that you're not calling those methods, but you are using the `role?('admin')` method. I would use one or the other and remove those that you don't use.